### DISCUSSION TITLE
How to select track in album view ( View #1)?

### DISCUSSION THREAD

**Discussion by Serif-7** [cmus/cmus#1075] [General]
I'm sorry if this isn't the right place to ask questions, but I couldn't find the answer in the cmus tutorial. In the album view I can of course select the album with the arrow keys, but I can't figure out how to switch the cursor to the track list on the right. Thanks in advance!

**Comment by pgaskin** [comment]
By default, you can press Tab to switch. If you do `:set mouse=1`, you can also use the mouse to do so.

> I'm sorry if this isn't the right place to ask questions

This is the correct place to ask questions.
